OpenEdXMobile/src/main/java/org/edx/mobile/deeplink/DeepLink.kt

```diff
@@ -18,36 +18,41 @@ open class DeepLink(val screenName: String) : Parcelable {
     var courseId: String? = null
     var componentId: String? = null
     var pathId: String? = null
-    var topicID: String? = null
-    var threadID: String? = null
+        get() = if (screenName == Screen.DISCOVERY_COURSE_DETAIL) {
```

Comment:
Please reorder the attributes to increase readability. Move `pathId` after `threadId`.

Reply:
done.